tox.ini

```diff
@@ -30,7 +30,23 @@ commands =
         --no-cov-on-fail \
         --cov-branch \
         --ignore-glob='tests/fixtures/*' \
-        --basetemp="{temp_dir}/pytest"
+        --basetemp="{temp_dir}/pytest" \
+        tests
+
+[testenv:py3_eest]
+extras =
+    test
+setenv =
+    COVERAGE_FILE = {toxworkdir}/.coverage
+commands =
+    pytest \
+        -c pytest-eest.ini \
+        -m "not slow and not zkevm" \
+        -n auto --maxprocesses 10 \
+        --basetemp="{temp_dir}/pytest" \
+        --clean \
```

Comment:
What does this argument do?

Reply:
cleans up any fixtures that may have been generated from previous runs. Otherwise pytest/fill will throw an error that there are already some fixtures. Not strictly required in the context of CI but is handy when someone is testing stuff locally